### ISSUE TITLE
Determine where markdown files will reside within the repository

### ISSUE THREAD

**Issue by zstix** [newrelic/docs-website#34]
## Description
In order to migrate content from the existing site to the new repository, we need to know where each page's markdown file will live within the repository. The original developer proposal was to have a directory structure that mimics the page's category. A page with this category structure `foo > bar > page` would be stored a directory like `src/md/foo/bar/page.mdx`.

Before we start the migration work, we need to confirm that this will work. If another structure is preferred, we should document that here to help inform the upcoming engineering work.

## Related Issue(s)
After reviewing the remaining research tasks, we have decided to break the work in #12 down into multiple smaller tickets. This is one of those tickets.

The developer proposal for this has been identified in #11. This ticket is to track that we still have an open decision to be made.

## Success Criteria
* [ ] Come to a consensus with engineering, PM, design, and other stakeholders for a first-pass solution to where we store content within the repository

**Comment by zstix** [comment]
There is some ongoing conversation about where we should put the files. Some of the options discussed include:
* Put the markdown files in a directory structure that maps to the existing category hierarchy
* Put the markdown files in a "shallow" version of the category hierarchy (i.e. all files in a directory for the top-level category)
* Put the markdown files in a completely flat directory structure (all in one or two directories)

The trade-off of a rigid system (like the first option) is that it's easier for contributors to find files to edit, but it's less flexible for where to put them.

Whatever we decide to do, we will always be flexible post-migration. Our setup in Gatsby doesn't really care where the files live within the `markdown-pages` directory.

We are planning on talking about this during our Monday sync and may schedule a follow-up meeting.

**Comment by zstix** [comment]
Upon further review, this seems like a duplicate of #11
